packages/console/src/SerialConsole/SerialConsole.js

```diff
@@ -1,11 +1,182 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const propTypes = {};
-const defaultProps = {};
+import { EmptyState } from 'patternfly-react';
+import { Button } from 'patternfly-react';
+import { bindMethods, noop } from 'patternfly-react';
+import { CONNECTED, DISCONNECTED, LOADING } from './constants';
 
-const SerialConsole = () => <div>Serial Console</div>;
+import XTerm from './XTerm';
+import SerialConsoleActions from './SerialConsoleActions';
 
-SerialConsole.propTypes = propTypes;
-SerialConsole.defaultProps = defaultProps;
+class SerialConsole extends React.Component {
+  constructor(props) {
+    super(props);
+
+    bindMethods(this, ['focusTerminal', 'onResetClick', 'onDisconnectClick']);
```

Comment:
We have deprecated `bindMethods` in favor of class properties.

Reply:
done